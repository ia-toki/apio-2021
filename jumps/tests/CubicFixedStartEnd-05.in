209636825ecd2c38299498cd26890ef71dd128c2
200 200
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 62 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 32 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191 192 193 194 195 196 197 198 199 200
0 0 199 199
59 59 176 176
20 20 118 118
82 82 190 190
175 175 185 185
86 86 188 188
66 66 155 155
30 30 148 148
154 154 164 164
10 10 51 51
3 3 83 83
48 48 158 158
138 138 169 169
34 34 92 92
19 19 111 111
48 48 105 105
178 178 190 190
46 46 194 194
74 74 87 87
27 27 129 129
65 65 91 91
54 54 173 173
27 27 198 198
181 181 194 194
172 172 179 179
84 84 164 164
15 15 182 182
178 178 185 185
84 84 92 92
14 14 170 170
105 105 156 156
190 190 193 193
95 95 128 128
94 94 152 152
89 89 160 160
159 159 186 186
1 1 142 142
146 146 181 181
88 88 132 132
2 2 117 117
89 89 190 190
90 90 181 181
51 51 85 85
17 17 94 94
33 33 182 182
138 138 146 146
159 159 197 197
71 71 93 93
172 172 183 183
142 142 199 199
107 107 157 157
83 83 129 129
142 142 171 171
144 144 199 199
97 97 121 121
110 110 130 130
35 35 106 106
34 34 50 50
102 102 185 185
198 198 199 199
16 16 160 160
72 72 111 111
154 154 168 168
26 26 113 113
88 88 159 159
196 196 199 199
59 59 142 142
187 187 191 191
157 157 198 198
84 84 102 102
192 192 196 196
25 25 46 46
139 139 156 156
46 46 179 179
76 76 167 167
90 90 94 94
161 161 198 198
69 69 106 106
10 10 37 37
186 186 190 190
29 29 48 48
55 55 63 63
139 139 176 176
27 27 179 179
108 108 151 151
140 140 197 197
29 29 133 133
97 97 148 148
179 179 186 186
19 19 125 125
141 141 185 185
125 125 190 190
193 193 196 196
90 90 142 142
135 135 191 191
95 95 132 132
115 115 184 184
46 46 54 54
137 137 153 153
198 198 199 199
54 54 71 71
92 92 177 177
142 142 194 194
165 165 192 192
171 171 188 188
50 50 117 117
63 63 150 150
99 99 137 137
70 70 106 106
152 152 156 156
109 109 197 197
29 29 182 182
174 174 196 196
44 44 191 191
62 62 150 150
124 124 141 141
45 45 110 110
16 16 106 106
137 137 152 152
52 52 176 176
146 146 196 196
177 177 186 186
124 124 148 148
32 32 143 143
107 107 184 184
118 118 157 157
179 179 183 183
174 174 197 197
119 119 173 173
51 51 164 164
1 1 192 192
109 109 184 184
192 192 198 198
30 30 88 88
17 17 112 112
107 107 167 167
129 129 177 177
7 7 129 129
137 137 154 154
97 97 109 109
127 127 162 162
98 98 103 103
137 137 172 172
5 5 167 167
121 121 189 189
193 193 197 197
71 71 182 182
168 168 191 191
91 91 183 183
11 11 148 148
84 84 152 152
131 131 148 148
133 133 194 194
146 146 174 174
53 53 130 130
144 144 185 185
165 165 173 173
83 83 111 111
33 33 92 92
160 160 198 198
112 112 120 120
172 172 173 173
145 145 164 164
156 156 185 185
97 97 114 114
80 80 191 191
109 109 174 174
77 77 129 129
191 191 199 199
113 113 119 119
12 12 47 47
136 136 187 187
49 49 155 155
97 97 102 102
49 49 120 120
121 121 177 177
196 196 197 197
74 74 107 107
153 153 198 198
72 72 153 153
150 150 165 165
158 158 185 185
58 58 178 178
61 61 180 180
141 141 177 177
157 157 173 173
197 197 198 198
86 86 194 194
6 6 56 56
74 74 141 141
190 190 198 198
140 140 198 198
163 163 176 176
41 41 50 50
55 55 121 121
95 95 152 152
118 118 140 140
22 22 150 150
60 60 172 172
27 27 83 83
