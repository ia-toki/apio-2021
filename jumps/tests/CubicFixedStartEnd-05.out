fc5005cd102fffc504bc1615010ebf88823d65f3
OK
169
116
68
108
10
102
89
88
10
-1
50
98
31
32
62
45
12
134
13
72
26
113
141
13
7
80
137
7
8
126
51
3
33
58
71
27
111
35
44
85
101
91
25
47
122
8
38
22
11
57
50
46
29
55
24
20
46
16
83
1
114
39
14
57
71
3
82
4
41
18
4
-1
17
119
91
4
37
37
-1
4
-1
3
37
122
43
57
74
51
7
76
44
65
3
52
56
37
69
8
16
1
11
85
52
27
17
57
87
38
36
4
88
123
22
131
88
17
50
60
15
116
50
9
24
83
77
39
4
23
54
104
161
75
6
28
65
60
48
92
17
12
35
5
35
132
68
4
111
23
92
107
68
17
61
28
70
41
8
28
32
38
8
1
19
29
17
111
65
52
8
6
-1
51
95
5
60
56
1
33
45
81
15
27
118
119
36
16
1
108
-1
67
8
58
13
9
61
57
22
98
111
26
